testutil/src/test/java/org/spine3/server/entity/EntityBuilderShould.java

```diff
@@ -84,6 +84,21 @@ public void create_entity() {
     }
 
     @Test
+    public void create_entity_with_default_values() {
+        final int version = 100500;
+        final String expectedState = Values.newStringValue("").getValue();
+        final Timestamp defaultTimestamp = Timestamp.getDefaultInstance();
+        final Entity entity = givenEntity().withVersion(version).build();
```

Comment:
If we try to create an `Entity` with the default values, why do we initialize it with a custom `version`?